Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -124,119 +162,226 @@ void Start()
             manipulationRecognizer.StartCapturingGestures();
         }
 
+        #region Interaction Management
+
+        /// <summary>
+        /// Thrown when we detect an interaction source.
+        /// </summary>
+        /// <param name="state"></param>
+        private void InteractionManager_SourceDetected(InteractionSourceState state)
+        {
+            trackedInteractionState.Add(state.source.id);
```

Comment:
yeah, you're correct. I was following the naming conventions from the previous pattern. Probably good to rename.